independent-projects/arc/runtime/src/main/java/io/quarkus/arc/impl/EventImpl.java

```diff
@@ -166,10 +166,11 @@ private Notifier<? super T> createNotifier(Class<?> runtimeType) {
 
     static <T> Notifier<T> createNotifier(Class<?> runtimeType, Type eventType, Set<Annotation> qualifiers,
             ArcContainerImpl container, InjectionPoint injectionPoint) {
-        return createNotifier(runtimeType, eventType, qualifiers, container, true, injectionPoint);
+        return createNotifier(runtimeType, eventType, qualifiers, container, !Arc.container().strictCompatibility(),
+                injectionPoint);
     }
 
-    static <T> Notifier<T> createNotifier(Class<?> runtimeType, Type eventType, Set<Annotation> qualifiers,
+    static <T> Notifier<T> createNotifier(Class<?> runtimeType, Type eventType, Set<Annotation> qualifiers, // TODO is this invoked somewhere
```

Comment:
We don't want to add more `TODO`s ;-). It's a package-private method so it shouldn't be hard to find out...

Reply:
Ah, remnant of a note to myself :D
Removed